python/ray/_private/external_storage.py

```diff
@@ -384,9 +383,7 @@ def __init__(
     def spill_objects(self, object_refs, owner_addresses) -> List[str]:
         if len(object_refs) == 0:
             return []
-        # Always use the first object ref as a key when fusing objects.
-        first_ref = object_refs[0]
-        filename = f"{first_ref.hex()}-multi-{len(object_refs)}"
+        filename = f"{uuid.uuid4().hex}-multi-{len(object_refs)}"
```

Comment:
Can we create a helper function to generate the unique filename so all ExternalStorage can share?